src/GovUk.Education.ExploreEducationStatistics.Content.Model/Methodology.cs

```diff
@@ -13,11 +13,12 @@ public class Methodology
 
         public List<PublicationMethodology> Publications { get; set; } = new();
 
-        [Required]
-        public string OwningPublicationTitle { get; set; }
+        [Required] public string OwningPublicationTitle { get; set; } = null!;
```

Comment:
Which formatting are we favouring?  Inline attributes or the other?

Reply:
I imagine I only changed this because Rider did it for me. I've changed them all to be inline just so they match - doesn't matter to me either way.